### PR TITLE
"Updated offensive behaviors, and…

### PR DESCRIPTION
**Pull request by ErfanFathii**
… modified KickDecisionMaker to use BhvStarterKickPlanner."

### CODE REVIEW

File: src/behaviors/bhv_starter_pass.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from src.interfaces.IBehavior import IBehavior
+from src.interfaces.IAgent import IAgent
+from pyrusgeom.soccer_math import *
+from pyrusgeom.geom_2d import *
+from src.utils.tools import Tools
+from service_pb2 import *
+
+
+class Pass(IBehavior):
```
Comment (naderzare) [inline src/behaviors/bhv_starter_pass.py]: class name should be BhvStarterPass

File: src/behaviors/bhv_starter_dribble.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from src.interfaces.IBehavior import IBehavior
+from src.interfaces.IAgent import IAgent
+from src.utils.tools import Tools
+from pyrusgeom.vector_2d import Vector2D
+from pyrusgeom.sector_2d import Sector2D
+from service_pb2 import *
+
+
+class Dribble(IBehavior):
```
Comment (naderzare) [inline src/behaviors/bhv_starter_dribble.py]: BhvStarterDribble

File: src/behaviors/bhv_starter_clearball.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+from src.interfaces.IBehavior import IBehavior
+from src.interfaces.IAgent import IAgent
+from pyrusgeom.soccer_math import *
+from pyrusgeom.geom_2d import *
+from src.utils.tools import Tools
+from pyrusgeom import vector_2d
+from service_pb2 import *
+
+class ClearBall(IBehavior):
```
Comment (naderzare) [inline src/behaviors/bhv_starter_clearball.py]: Bhv...

File: src/decision_makers/kick_decision_maker.py [hunk 4]
```diff
@@ -10,10 +10,11 @@
...
     def __init__(self):
         self.bhv_kick_planner = BhvKickPlanner()
-        # self.bhv_kick_planner = BhvStarterKickPlanner()
+        self.bhv_kick_planner = BhvStarterKickPlanner()
 
     def make_decision(self, agent: IAgent):
         agent.logger.debug("--- WithBallDecisionMaker ---")
         from src.sample_player_agent import SamplePlayerAgent  # Local import to avoid circular import
         assert isinstance(agent, SamplePlayerAgent)
+        #self.bhv_kick_planner.execute(agent)
```
Comment (naderzare) [inline src/decision_makers/kick_decision_maker.py]: we do not need this line

File: src/decision_makers/kick_decision_maker.py [hunk 5]
```diff
@@ -10,10 +10,11 @@
 class KickDecisionMaker(IDecisionMaker):
     def __init__(self):
         self.bhv_kick_planner = BhvKickPlanner()
-        # self.bhv_kick_planner = BhvStarterKickPlanner()
+        self.bhv_kick_planner = BhvStarterKickPlanner()
```
Comment (naderzare) [inline src/decision_makers/kick_decision_maker.py]: for default we should not use Starter classes, but we can have a variable in sample player agent, for example starter=true/false and we can choose planner or starter planner based on that variable

**Review by naderzare — CHANGES_REQUESTED** [review]

**Review by SoroushMazloum — COMMENTED** [review]
I fixed these issues in my new commit

**Review by SoroushMazloum — APPROVED** [review]

**Review by SoroushMazloum — APPROVED** [review]

**Pull request closed without merging**